tensorrt_llm/_torch/visual_gen/mapping.py

```
@@ -34,9 +35,10 @@ class VisualGenMapping(DeviceMeshTopologyImpl):
 
     Parallelism Hierarchy:
         total_workers = cfg × sp
-        sp (sequence parallelism) = cp × ulysses  [mutually exclusive today; TODO to combine]
-        cp (context parallelism)  = ring           [ring attention, not yet implemented]
-                                  | attn2d         [Attention2D 2D mesh, row_size × col_size]
+        sp (sequence parallelism) = cp × ulysses
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)